aws-lc-rs/README.md

```diff
@@ -70,6 +70,30 @@ the pre-generated bindings. This feature requires `libclang` to be installed. Se
 [requirements](https://rust-lang.github.io/rust-bindgen/requirements.html)
 for [rust-bindgen](https://github.com/rust-lang/rust-bindgen)
 
+##### prebuilt-nasm
+
+For Windows x86 and x86-64, the AWS-LC assembly code requires NASM to build. On these platforms, if a NASM assembler is
+found during the build process, then it *will* be used to compile the assembly files. However, if a NASM assembler is
+not found, then the build may fail except in the following cases:
```

Comment:
I would add a sentence recommending that people install NASM instead of using this flag as an easy fix.